sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/NewIssue.java

```diff
@@ -86,4 +86,11 @@ public interface NewIssue {
    */
   void save();
 
+  /**
+   * Register a specific context for this issue.
+   * e.g. "Spring"
+   * @since 9.6
+   */
```

Comment:
```suggestion
  /**
   * Set the optional rule description section context key. 
   * The key need to match the one present in {@link Context#getKey()}.
   * @since 9.6
   */
```